docs/index.md

```diff
@@ -15,13 +15,13 @@ We have four key values in which we stand by:
 3. **People-first**: As a people business, we place your customers and employees at the centre of any digital project or transformation. People are at the heart of everything we design and deliver.
 4. **Pragmatic**: All technology has its strengths but also its limitations. We focus on delivering solutions that work and on creating technology ecosystems that make the most of the technologies within them. We are honest about the value that can be delivered now but also work to ensure our solutions have the agility and scalability built in to meet the challenges of tomorrow.
 
-## Why Stacks
+## Why Stacks?
```

Comment:
This isn't valid markdown heading, hence why the listing failed. Either we make a statement, eg. `Why we implemented Stacks` or we turn off that listing rule for this file?